src/code/bank1.asm

```diff
@@ -2005,7 +2005,7 @@ CreateFollowingNpcEntity::
     and  a                                        ; $6022: $A7
     jr   nz, .ghostEnd                            ; $6023: $20 $5A
     ldh  a, [hMapRoom]                            ; $6025: $F0 $F6
-    cp   $40                                      ; $6027: $FE $40
+    cp   IS_UNKWON_ROOM_CATEGORY                  ; $6027: $FE $40
```

Comment:
This doesn't check for a specific room, but causes the ghost to follow you when you leave the upper 25% of the overworld map after finishing dungeon 4.